changelog.md

```diff
@@ -3,7 +3,145 @@
 SFML 3 is coming soon!
 For a sneak peak at what has changed, check out the [migration guide](migration.md).
 
-## Unreleased SFML 2.6.2
+## Unreleased SFML 3.0.0
+
+### General
+
+**Features**
+
+-   Raised C++ requirement to C++17 (#1855)
+-   Modernized internals to the full extent allowed by C++17
+-   Upgraded to CMake 3.22 (#2543)
+-   Added many unit tests with >50% code coverage
+-   Added many new CI jobs for a huge variety of platforms and configurations
+-   Added clang-tidy config
+-   Added clang-format config
+-   Removed OpenAL shared library requirement (#2749)
```

Comment:
This is mentioned again in the "Audio" section but I thought it was worth highlighting the removal of OpenAL near the top. That I don't expect people to pay as much attention after the first 10 bullet points.